README.md

```diff
@@ -1,25 +1,28 @@
-# djangocms-cascade
+djangocms-cascade [![Latest version on PyPI](https://img.shields.io/pypi/v/djangocms-cascade.svg)](https://pypi.python.org/pypi/djangocms-cascade)
+==================================================================================================================================================
 
-**DjangoCMS-Cascade** is the Swiss army knife for working with Django CMS plugins.
+[![Build Status](https://travis-ci.org/jrief/djangocms-cascade.png?branch=master)](https://travis-ci.org/jrief/djangocms-cascade)
+[![Code health](https://landscape.io/github/jrief/djangocms-cascade/master/landscape.svg?style=flat)](https://landscape.io/github/jrief/djangocms-cascade/master)
+[![Python versions](https://img.shields.io/pypi/pyversions/djangocms-cascade.svg)](https://pypi.python.org/pypi/djangocms-cascade)
+[![Monthly downloads from PyPI](https://img.shields.io/pypi/dm/djangocms-cascade.svg)](https://pypi.python.org/pypi/djangocms-cascade)
+[![Software license](https://img.shields.io/pypi/l/djangocms-cascade.svg)](https://github.com/jrief/djangocms-cascade/blob/master/LICENSE-MIT)
+[![Gitter chat room](https://badges.gitter.im/jrief/djangocms-cascade.svg)](https://gitter.im/jrief/djangocms-cascade)
 
-## NEWS for version 0.9
+**DjangoCMS-Cascade** is the Swiss army knife for working with Django CMS plugins.
 
-* Compatible with django-CMS 3.3.0
-* Important notice: Converted ``SharableCascadeElement`` into a proxy model. A migration
-  merges these two models, so please backup your database before upgrading.
-* Add support for Section Bookmarks. This allows to add links on subsections of a CMS page.
-* Fixes many minor issues, check the changelog for details.
-* Replaced configuration settings ``CMSPLUGIN_CASCADE_LINKPLUGIN_CLASSES`` against
-  ``CMSPLUGIN_CASCADE['link_plugin_classes']`` for better consistency.
```

Comment:
I think we should not delete this information. Can you please re-add these lines? Then we can probably merge this. Thanks!